style/middleware/_middleware.quizactivity.scss

```diff
@@ -41,4 +41,45 @@
 .answeroption .row .checkbox label p, .answeroption .row .checkbox label {
   display: inline-block;
   font-weight: normal;
+  float: left;
+  width: 95%;
+}
+
+.answeroption .row .checkbox label input[type='checkbox'] {
+  float: left;
+  margin-top: $global-spacing-unit-tiny;
+  margin-left: $global-spacing-unit-tiny;
+}
+
+.answer label {
+  display: inline-block;
+  margin-left: $global-spacing-unit-tiny;
+  font-weight: normal;
```

Comment:
Is this overriding a previous setting? Can we prevent this somehow?

Reply:
There was no setting to this selector initially and the text was showing up as bold and not aligned with checkboxes. In order to align them, I had to add some of these styles. Making the text non-bold just makes it look neater but can remove that bit if you like.